contracts/deploy-history/poc-polygon-2021-09-23T23:52:54.769Z.json

```diff
@@ -0,0 +1,30 @@
+{
+  "deployer": "0xD2553382a60F121d9b1e35cFC9EBF4870FbCC96F",
+  "config": {
+    "donationToken": "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063",
+    "uniswapFactory": "0xc35DADB65012eC5796536bD9864eD8773aBc74C4",
+    "weth": "0x0d500B1d8E8eF31E21C99d1Db9A6444d3ADf1270",
+    "metadataAdmin": "0xD2553382a60F121d9b1e35cFC9EBF4870FbCC96F",
+    "payoutAdmin": "0xD2553382a60F121d9b1e35cFC9EBF4870FbCC96F",
+    "matchingToken": "0x8f3Cf7ad23Cd3CaDbD9735AFf958023239c6A063",
+    "roundStartTime": 1632442000,
+    "roundEndTime": 1663200000,
+    "ipfsRetrievalEndpoint": "https://ipfs-dev.fleek.co/ipfs",
+    "roundLogoPath": "./scripts/config/assets/dgrants-placeholder.png",
+    "metadataJson": {
+      "name": "dGrants Proof-of-Concept",
+      "description": "Gitcoin building Gitcoin using Gitcoin 🤯",
+      "grants": [0, 1, 2, 3, 4, 5],
+      "logoURI": "https://ipfs-dev.fleek.co/ipfs/QmYMXFkv1LU2YzQPe8LEDR7NrtL1rgNXV5un9Fn9hH765F"
+    }
+  },
+  "contracts": {
+    "GrantRegistry": "0x0D39B9BB82d41F33F894FCeb85FbE4777377A092",
+    "GrantRoundManager": "0x6C9C48aE6A74DA5A98CFAD6617cF0B860E112071"
+  },
+  "actions": {
+    "PublishGrantRoundLogo": "QmYMXFkv1LU2YzQPe8LEDR7NrtL1rgNXV5un9Fn9hH765F",
+    "PublishGrantRoundMetadata": "QmPsAXpuwddVQ2w5jfGFZSSW1oTNQCoSJPhpAfhLXyZjUN",
+    "CreateGrantGround": "0xd277a2d9197cf4ddc9985430928f2a611c1fc4ab29c1c8bbc7fa728c83247d7b"
+  }
+}
```

Comment:
^ remove this ?

Reply:
Done in 04468b29d8299256b55fd1c608f882bda66974f0